packages/shared-components/src/Forms/SubmissionWrapper.jsx

```diff
@@ -3,7 +3,7 @@ import { Redirect } from "react-router-dom";
 
 export const SubmissionWrapper = ({ submission, url, children }) => {
   if (!submission) {
-    return <Redirect to={url} />;
+    return <Redirect to={`${url}${window.location.search}`} />;
```

Comment:
bruke useLocation() i stedet for window.location?

Reply:
byttet ut med useLocation()